### PR TITLE
ref(crons): Limit Monitor.status to ONLY object status

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Review by rjo100 — DISMISSED** [review]
are we going to migrate/remove `get_status_display`? or just leave everything in the db as is

File: src/sentry/monitors/models.py [hunk 1]
```diff
@@ -176,7 +190,7 @@ class Monitor(Model):
     project_id = BoundedBigIntegerField(db_index=True)
     name = models.CharField(max_length=128)
     status = BoundedPositiveIntegerField(
-        default=MonitorStatus.ACTIVE, choices=MonitorStatus.as_choices()
+        default=ObjectStatus.ACTIVE, choices=ObjectStatus.as_choices()
```
Comment (rjo100) [inline src/sentry/monitors/models.py:193]: ah wait we probably need a migration here

Comment (evanpurkhiser) [reply src/sentry/monitors/models.py:193]: Already done in https://github.com/getsentry/sentry/pull/48230

Comment (rjo100) [reply src/sentry/monitors/models.py:193]: ah got it. ok so let's remove `get_status_display` then and we're good ot go

**Review by rjo100 — APPROVED** [review]

**Pull request merged**